I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 76, in script
    $ renpy.pause(0.5)
  File "game/script.rpy", line 76, in <module>
    $ renpy.pause(0.5)
Exception: Required parameter what has no value.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 76, in script
    $ renpy.pause(0.5)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\python.py", line 1641, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 76, in <module>
    $ renpy.pause(0.5)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\exports.py", line 1267, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ui.py", line 278, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 2496, in interact
    repeat, rv = self.interact_core(preloads=preloads, **kwargs)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 2764, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 464, in visit_all
    d.visit_all(callback)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 464, in visit_all
    d.visit_all(callback)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 464, in visit_all
    d.visit_all(callback)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\screen.py", line 394, in visit_all
    callback(self)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\core.py", line 2764, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\screen.py", line 404, in per_interact
    self.update()
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\screen.py", line 573, in update
    self.screen.function(**self.scope)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 148, in apply_arguments
    return parameters.apply(args, kwargs, ignore_errors)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 117, in apply
    raise Exception("Required parameter %s has no value." % name)
Exception: Required parameter what has no value.

Windows-8-6.2.9200
Ren'Py 6.99.11.1749
Yuri Jam 1.0
